assets/templates/mail/comment_notification.txt

```diff
@@ -0,0 +1,10 @@
+{% load settings %}
+
+{{ comment.poster }} posted the following comment:
+
+{{ comment.message }}
+
+You can respond to {{ comment.poster }} by replying to this e-mail.
```

Comment:
Damn, nice function. How did you set that up?

Reply:
I'm not doing fancy stuff :blush:, just keeping it simple. The sender e-mail is added in the Reply-To header [here](https://github.com/VSEScala/Scala-Dining/pull/282/files#diff-2608dfa2b591f048a4302446578a26fd55e8062c86e25c6b1cdafd20eed71d98R587).
It's mentioned under the checkbox that the sender's e-mail will be included in the message, see the screenshot. That also kind of acts as a disincentive for spam, because the receiver gets to know the sender's e-mail.

Although I just realize that they could use a unverified e-mail address on the site. But as long as that's not being misused we can keep it like this.